Controller/ResettingController.php

```diff
@@ -73,51 +73,42 @@ public function sendEmailAction(Request $request)
             return $event->getResponse();
         }
 
-        if (null === $user) {
-            return $this->render('FOSUserBundle:Resetting:request.html.twig', array(
-                'invalid_username' => $username
-            ));
-        }
+        if (null !== $user && !$user->isPasswordRequestNonExpired($this->container->getParameter('fos_user.resetting.token_ttl'))) {
```

Comment:
looks like this line exceeds 120 chars, or didn't we respect this in this bundle?